cmd/influxd/main.go

```diff
@@ -112,9 +106,6 @@ func execRun(args []string) {
 	defer stopProfiling()
 
 	// Print sweet InfluxDB logo and write the process id to file.
-	fmt.Print(logo)
```

Comment:
As discussed, you may like to leave this in. With the recent logging fixes, the logo only goes to stdout, and stdout goes to `/dev/null` when influxd is launched as a service. When launched manually it is printed, which may be useful to people.